apps/nextra/pages/en/build/advanced-guides/randomness.mdx

```diff
@@ -114,6 +114,8 @@ where:
 - `let winner_idx = aptos_framework::randomness::u64_range(0, n);` is the a randomness API call that returns a u64 integer in range `[0, n)` uniformly at random.
 - `#[randomness]` is a required attribute to enable the API call at runtime.
 
+Note: there are security nuances to consider, explained in more details in "Security Considerations" section. Compiler helps with test and abort attacks by requiring functions using randomness to be private. On the other hand, randomness API currently doesn’t prevent undergassing attacks, and so code needs to be written in a specific way to avoid it.
```

Comment:
```suggestion
<Callout type="info">
### Security Considerations
Compiler helps with test and abort attacks, requiring functions using randomness to be private.  However, the randomness API current does not prevent undergassing attacks.  The smart contract will be need to be written in a certain way to avoid it.
</Callout>
```